dakota/synergistic/fs-ty/fs-ty.py

```diff
@@ -0,0 +1,105 @@
+import external_cym
+import multiprocessing
+import sys
+import os
+import signal
+import dakota.interfacing as di
+import subprocess
+import glob
+sys.path.append('../../../scripts')
+import input as inp
+import output as oup
+
+cycdir = '../../../cyclus-files/synergistic/fs-ty/'
+
+# ----------------------------
+# Parse Dakota parameters file
+# ----------------------------
+params, results = di.read_parameters_file()
```

Comment:
Why is so much of this in global scope?

Reply:
This is how the dakota tutorial for python interfacing sets it up